packages/pringo/pringo.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+synopsis: "Pseudo-random, splittable number generators"
+description:
+  "Pseudo-random number generators that support splitting and two interfaces: one stateful, one purely functional"
+maintainer: "Xavier Leroy <xavier.leroy@college-de-france.fr>"
+authors: "Xavier Leroy <xavier.leroy@college-de-france.fr>"
+license: "GPL v2 with static linking exception"
+homepage: "https://github.com/xavierleroy/pringo"
+bug-reports: "https://github.com/xavierleroy/pringo/issues"
+depends: [
+  "ocaml" {>= "4.05.0"}
+  "ocamlfind"
+]
+build: make
+install: [make "install"]
+remove: [make "uninstall"]
```

Comment:
This can be removed with opam2

Reply:
It wasn't clear to me how OPAM2's automatic uninstall deals with ocamlfind-installed libraries.

Reply:
Opam diffs the file tree in the `.opam` folder before and after the installation, and notes down the changes. At uninstall stage it reverts those changes: the `remove` stanza is still useful if you are making changes to the content of pre-existing files that need to be reverted during the uninstall phase.

Reply:
Thanks for the explanations.  I removed the "remove" line (but managed to botch the log entry).